resources/scss/main.scss

```diff
@@ -0,0 +1,63 @@
+@import "footer.scss";

```

Comment:
the order of import is not correct:
. reset.scss
. global.scss
. module.scss
. page.scss

Reply:
done